pkg/controllers/cluster/status.go

```diff
@@ -121,7 +122,11 @@ func (cc *ClusterReconciler) updateStatus(ctx context.Context, cluster *scyllav1
 					return errors.New("seed node replace is not supported")
 				}
 				if replaceAddr == "" {
-					rackStatus.ReplaceAddressFirstBoot[svc.Name] = svc.Spec.ClusterIP
+					memberIp, err := resource.GetIpFromService(&svc, cluster)
+					if err != nil {
+						return err
```

Comment:
I'm wondering if we shouldn't log info/debug and skip this service instead of returning an error. With current code, we won't update other services until pod is running.

Reply:
It doesn't seem fine to me to not throw an issue in that stage
If we reach replacement state for a node it means it should have an IP set so this condition should not happen unless there is a bug or unavailability of k8s
We rely on that status then to be able to replace the pod. Also if we skip it there is a high chance that no other events will lead to reconcile request which means potentially having a pod stuck